packages/modeling/src/geometries/poly3/measureArea.js

```diff
@@ -14,19 +15,18 @@ const measureArea = (poly3) => {
   }
   const vertices = poly3.vertices
 
-  // calculate a real normal
-  const a = vertices[0]
-  const b = vertices[1]
-  const c = vertices[2]
-  const ba = vec3.subtract(vec3.create(), b, a)
-  const ca = vec3.subtract(vec3.create(), c, a)
-  const normal = vec3.cross(ba, ba, ca)
+  // calculate a normal vector
+  const normal = plane.fromPoints(plane.create(), ...vertices)
```

Comment:
I think that calling poly3.plane() would be better as this function caches the plane, and will return the plane if already available. It might help performance.

Reply:
Done